utoipa-gen/tests/testdata/schema_generic_collect_non_inlined_schema

```diff
@@ -140,7 +140,7 @@
                 },
                 "t": {
                   "type": "integer",
-                  "format": "int32"
+                  "format": "int64"
```

Comment:
These JSON changes seems correct except how this `int32` changes to `int64`

Reply:
`FooStruct_HashMap_i32_Person_i64` is the schema for `HashMap(FooStruct<std::collections::HashMap<i32, Person<i64>>>)`. So `Person<i64>::t` is typed as `i64`. No?

Reply:
Ah yes, true. :smile:

Reply:
I think this is solved, and will be fixed by changing the default implementations.